presets/okta/syslog/preset.yaml

```diff
@@ -0,0 +1,695 @@
+                WHEN LOWER(eventType) = 'system.import.implicit_deletion.complete' THEN 'Entity Management'
+                WHEN LOWER(eventType) = 'system.import.implicit_deletion.start' THEN 'Entity Management'
+                WHEN LOWER(eventType) = 'system.import.membership_processing.complete' THEN 'Group Management'
+                WHEN LOWER(eventType) = 'system.import.membership_processing.start' THEN 'Group Management'
+                WHEN LOWER(eventType) = 'system.import.object_creation.complete' THEN 'Entity Management'
+                WHEN LOWER(eventType) = 'system.import.object_creation.start' THEN 'Entity Management'
+                WHEN LOWER(eventType) = 'system.import.start' THEN 'Account Change'
+                WHEN LOWER(eventType) = 'system.import.user.complete' THEN 'Account Change'
+                WHEN LOWER(eventType) = 'system.import.user.update' THEN 'Account Change'
+                WHEN LOWER(eventType) = 'system.import.user_matching.complete' THEN 'Account Change'
+                WHEN LOWER(eventType) = 'system.import.user_matching.start' THEN 'Account Change'
+                WHEN LOWER(eventType) = 'system.operation.concurrency_limit.violation' THEN 'Entity Management'
+                WHEN LOWER(eventType) = 'system.operation.rate_limit.violation' THEN 'Entity Management'
+                WHEN LOWER(eventType) = 'system.push.send_factor_verify_push' THEN 'Authentication'
+                WHEN LOWER(eventType) = 'user.account.update_profile' THEN 'Account Change'
+                WHEN LOWER(eventType) = 'user.authentication.auth_via_mfa' THEN 'Authentication'
+                WHEN LOWER(eventType) = 'user.authentication.sso' THEN 'Authentication'
+                WHEN LOWER(eventType) = 'user.authentication.verify' THEN 'Authentication'
+                WHEN LOWER(eventType) = 'user.lifecycle.deactivate' THEN 'Account Change'
+                WHEN LOWER(eventType) = 'user.lifecycle.suspend' THEN 'Account Change'
+                WHEN LOWER(eventType) = 'user.mfa.factor.activate' THEN 'Authentication'
+                WHEN LOWER(eventType) = 'user.mfa.factor.update' THEN 'Authentication'
+                WHEN LOWER(eventType) = 'user.session.access_admin_app' THEN 'Authentication'
+                WHEN LOWER(eventType) = 'user.session.clear' THEN 'Authorize Session'
+                WHEN LOWER(eventType) = 'user.session.start' THEN 'Authorize Session'
+                ELSE NULL
+            END
+        - name: ocsf_activity_id
+          expr: |
+            CASE
+                WHEN LOWER(eventType) = 'app.user_management.grouppush.mapping.okta.users.ignored' THEN CAST('3' AS INT)
+                ELSE CAST('99' AS INT)
+            END
+        - name: ocsf_activity_name
+          expr: |
+            CASE
+                WHEN LOWER(eventType) = 'app.user_management.grouppush.mapping.okta.users.ignored' THEN 'update'
+                ELSE 'other'
+            END
```

Comment:
I'm worried about this one...  I would say normally it looks like a little suspicious for all of the event types to go into "other".  I spot checked a random few and I think there may be arguments for different activity names.